style.css

```diff
@@ -97,3 +98,225 @@ ul {
+  display: flex;
+  width: 100%;
+  height: auto;
+}
+
+.stories {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  align-self: flex-start;
+}
+
+.about-stories {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.cards {
+  display: grid;
+  column-gap: 12px;
+  grid-template-columns: 1fr 1fr 2fr 1fr;
+  padding-inline-start: 0;
+}
+
+.cards li {
+  border: 1px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 12px 10px 12px;
+}
+
+.link-pro {
+  text-decoration: none;
+  color: white;
+  font-size: 17px;
+  align-self: flex-start;
+}
+
+.see-pro {
+  border: 1px solid #ff6b00;
+  background: #ff6b00;
+  padding: 12px;
+  font-family: 'Inter', sans-serif;
+  text-align: center;
+}
+
+.second-project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url(images/imgbackground.png);
+  padding: 24px;
+  margin-top: 24px;
+}
+
+.project-heading {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  color: white;
+  align-self: flex-start;
+}
+
+.description {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  color: #fff;
+}
+
+.cards-project2 {
+  display: grid;
+  column-gap: 12px;
+  grid-template-columns: 1fr 2fr 1fr;
+  padding-inline-start: 0;
+}
+
+.cards-project2 li {
+  border: 1px solid white;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px 12px 10px 12px;
+  color: white;
+  background-color: rgba(255, 255, 255, 0.24);
+}
+
+.link-project2 {
+  text-decoration: none;
+  color: white;
+  font-size: 17px;
+  width: 100%;
+}
+
+.see-project2 {
+  border: 1px solid #ff6b00;
+  background: #ff6b00;
+  padding: 12px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #1c1a19;
+  width: 100%;
+}
+
+.about-me-top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px 120px 24px;
+}
+
+.about-header {
+  font-family: 'Crete Round', serif;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  font-weight: 400;
+  color: #ff6b00;
+}
+
+.description-about-me {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  font-weight: 400;
+  color: white;
+}
+
+.resumi-action {
+  display: flex;
+  padding-left: 24px;
+}
+
+.victor-about-me {
+  display: flex;
+  position: absolute;
+  right: 2px;
+}
+
+.about-me-bottom {
+  margin-top: 100px;
+}
+
+.skills {
+  color: white;
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 500;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 70px;
+  width: 100%;
+  padding: 120px 24px;
+}
```

Comment:
- [ ] Kindly align these to the center in order to match your design to the Figma design. Please work on this 👍 

Your design
![image](https://user-images.githubusercontent.com/50941074/159894563-80e09b60-c2ed-4c6f-8e10-5842b2efe172.png)

The Figma design
![image](https://user-images.githubusercontent.com/50941074/159894660-16c60424-d3ca-4404-a6e6-674f36870668.png)